pkgs/development/python-modules/coordinates/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "coordinates";
+  version = "0.4.0";
+
```

Comment:
Please set `disabled = !isPy3k;`.

Reply:
Sorry, also needs to be disabled for Python 3.5.

Reply:
`disabled = pythonOlder "3.5";`